kong/templates/kong_defaults.lua

```diff
@@ -21,6 +21,7 @@ admin_listen = 127.0.0.1:8001 reuseport backlog=16384, 127.0.0.1:8444 http2 ssl
 status_listen = off
 cluster_listen = 0.0.0.0:8005
 cluster_control_plane = 127.0.0.1:8005
+cluster_protocol = NONE
```

Comment:
https://github.com/Kong/kong/blob/7fe16e6330d9525d8ecd154f573282a24c5c41ab/kong/conf_loader/init.lua#L643-L652

https://github.com/Kong/kong/blob/7fe16e6330d9525d8ecd154f573282a24c5c41ab/kong/db/declarative/init.lua#L401-L404

https://github.com/Kong/kong/blob/7fe16e6330d9525d8ecd154f573282a24c5c41ab/CHANGELOG.md?plain=1#L124-L125

https://github.com/Kong/kong/blob/7fe16e6330d9525d8ecd154f573282a24c5c41ab/kong.conf.default#L190-L193

These four files should be updated if we allow users to specify the CP/DP protocol.